include/conversation.php

```diff
@@ -771,7 +771,7 @@ function conversation_add_children(array $parents, $block_authors, $order, $uid)
 	$items = [];
 
 	foreach ($parents AS $parent) {
-		$condition = ["`item`.`parent-uri` = ? AND `item`.`uid` IN (0, ?) AND `vid` != ?",
+		$condition = ["`item`.`parent-uri` = ? AND `item`.`uid` IN (0, ?) AND `vid` != ? ",
```

Comment:
This should be fixed centrally in the Database driver instead, this behavior is neither desirable not documented. It's error-prone like this fix shows and isn't sustainable on the long term.

Reply:
This is no issue of the database driver. Later in the code we add some more stuff to the condition: https://github.com/friendica/friendica/blob/develop/include/conversation.php#L802

Either I add the space here or there.